tests/base/boring_model.py

```diff
@@ -76,6 +76,7 @@ def training_step(...):
         """
         super().__init__()
         self.layer = torch.nn.Linear(32, 2)
+        self.example_input_array = torch.rand(5, 32)
```

Comment:
let’s keep boring model, boring. 

can you remove the example array from here?

then in each test that needs this, subclass boring model and add the example array or set it after init

Reply:
I though we can have this since it's not changing any methods inside `BoringModel`. Will update this then.